test/serde_arrow_primitive_array_SUITE.erl

```diff
@@ -0,0 +1,58 @@
+-module(serde_arrow_primitive_array_SUITE).
+
+-compile(export_all).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("stdlib/include/assert.hrl").
+
+-include("serde_arrow_primitive_array.hrl").
+
+all() ->
+    [
+        valid_len_on_new,
+        valid_null_count_on_new,
+        valid_validity_bitmap_on_new,
+        valid_len_on_access,
+        valid_null_count_on_access,
+        valid_validity_bitmap_on_access
```

Comment:
I don't like these names. Is there a convention I could follow (in CommonTest tests in general)?

Reply:
Na, no real convention. I'd probably combine these into like a test of `primitive_array_creation_tests` and just do all the tests for that in the one function.